### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by mumo-esther**
- The following tasks were accomplished in this feature:

  - Implemented comments controller & methods
  - Implemented likes controller & `create_user` like method
  - Implemented `create_user` post view
  - Update config `routes.rb`
  - Add partial for flash

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = current_user.comments.new(comment_params)
```
Comment (brytebee) [inline app/controllers/comments_controller.rb:7]: Kindly define a `new` action/method for this controller. Your line 3 uses the new method which is currently undefined.

Consider this snippet.

```ruby
def new
   # code
end
```

File: config/routes.rb [hunk 2]
```diff
@@ -2,6 +2,9 @@
     root "users#index"
     
     resources :users, only: [:index, :show] do
-        resources :posts, only: [:index, :show]
+      resources :posts, only: [:index, :new, :create, :show] do
+       resources :comments, only: [:create]
```
Comment (brytebee) [inline config/routes.rb]: Update the comments resource to accommodate your newly created `new` method.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @mumo-esther👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good understanding and implementation of routing.✔️
- Great mapping of data from the controller actions to the views.✔️
- Great DB design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @mumo-esther 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**